probe/kubernetes/deployment.go

```diff
@@ -46,9 +46,13 @@ func (d *deployment) Selector() (labels.Selector, error) {
 }
 
 func (d *deployment) GetNode(probeID string) report.Node {
+	desiredReplicas := 1
+	if d.Spec.Replicas != nil {
+		desiredReplicas = int(*d.Spec.Replicas)
+	}
```

Comment:
yes, I picked it up in passing while fixing the exact same issue in StatefulSets. It was easier to not make a seperate PR.